teps/0091-trusted-resources.md

```diff
@@ -254,47 +276,52 @@ metadata:
   name: config-trusted-resources
   namespace: tekton-pipelines
 data:
-  onError: "stopAndFail"
+  verify: true
   cosign-pubkey-path: "/etc/signing-secrets/cosign.pub"
   kms-pubkey-path: "gcpkms://projects/<project>/locations/<location>/keyRings/<keyring>/cryptoKeys/<key>"
 ```
-`onError: [ stopAndFail | continue ]`. (Optional, default `continue`)
-`stopAndFail` will cause the build to fail if image verification fails,  `continue` will allow the build to continue even
-if verification fails.
 
-`skip-verification`. (Optional, default `true`), if true then directly return nil, if false then do the verfication
+`verify`. (Optional, default `false`), if false then directly return nil, if true then do the verification
 
 `cosign-pubkey-path`. (Optional, default `/etc/signing-secrets/cosign.pub`), it specifies the secret path to store the cosign pubkey
 
 `kms-pubkey-path`.  (Optional, default empty), it specifies the KMS reference
 
-This will allow the configuration for one key per key type. But we cannot config multiple cosign keys or kms keys.
-To address this
+The key path configuration can be used as default keys when no keys are found in `keyref`.
 
 
 ### Integrate with Remote Resource Resolution
 
 TEP-0060 introduces a new ResourceRequest reconciler that will actually pull
 the remote image and expand it to a Task.
 
-There are several ways to integrate the Trusted Tesource with Remote Resource Resolution
+This TEP proposes creating the remote resolution request before verification in webhook. Alternatives will be discussed at [alternatives](#alternatives).
 
 
-| Method | Pros | Cons |
-| -------- | ----------- | ----------- |
-| Fetch remote resources in validating Webhook | Verification fail fast in webhook | Duplicate work for Resolution
-| Verify in Controller | No duplicate work for Resolution | The verification cannot fail fast in webhook. The resources may have been stored in etcd and used by other components
-| Verify in Remote Resolution | No duplicate work for Resolution | Verification coupled with Resolution
+## Threat Models
```

Comment:
I add this section and want to discuss how this TEP can help to secure the potential threats. 
Please take a look and let me know if they make sense. Thanks!
@wlynch @dibyom